packager/CMakeLists.txt

```diff
@@ -227,10 +227,27 @@ add_custom_target(packager_test_py_copy ALL
 )
 
 if(NOT SKIP_INTEGRATION_TESTS)
+  if(NOT Python3_EXECUTABLE)
+    find_package(Python3 COMPONENTS Interpreter REQUIRED)
+  endif()
+
   add_test (NAME packager_test_py
-    COMMAND ${PYTHON_EXECUTABLE} packager_test.py
+    COMMAND ${Python3_EXECUTABLE} packager_test.py
```

Comment:
Could you please also use `${Python3_EXECUTABLE}` in `packager/tools/CMakeLists.txt` and `packager/version/CMakeLists.txt` in place of `python3`?  I think that might solve https://github.com/shaka-project/shaka-packager/issues/1385